BigQuery/tests/System/LoadDataAndQueryTest.php

```diff
@@ -158,6 +159,60 @@ public function testRunQuery($useLegacySql)
         }
     }
 
+    public function testInsertRowToTableWithDefaultValueExpression()
+    {
+        // unset default value expression fields so that they are populated automatically
+        unset(
+            $this->row['Name'],
+            $this->row['Age'],
+            $this->row['Weight'],
+            $this->row['IsMagic'],
+            $this->row['CreatedTimestamp']
+        );
```

Comment:
I would suggest setting this to a local variable before unsetting it.
```suggestion
        $row = $this->row;
        unset(
            $row['Name'],
            $row['Age'],
            $row['Weight'],
            $row['IsMagic'],
            $row['CreatedTimestamp']
        );
        $insertResponse = self::$table->insertRow($row);
```